pom.xml

```diff
@@ -167,8 +167,20 @@
       <groupId>org.folio</groupId>
       <artifactId>data-import-processing-core</artifactId>
       <version>2.1.6</version>
+      <exclusions>
```

Comment:
Can we try it with this exclusion alone? I think it should work without the added javax.mail dependency as it worked before.